API/Controllers/MemoryUpdateController.js

```diff
@@ -0,0 +1,22 @@
+import {updateMemory} from '../Helpers/query';
+
+class MemoryUpdateController {
+
+    static updateMemory(request, response) {
+
+        const id = request.body.id
```

Comment:
You don't need to alter the id, The id is already passed in from the endpoint url, in `/:id`. Consider removing.
To pick that
```suggestion
        const id = request.params.id
```